app/src/main/java/com/movies/presentation/base/fragment/BaseFragment.kt

```diff
@@ -26,7 +32,8 @@ abstract class BaseFragment<T : Any, VM : BaseViewModel<T>> : Fragment(), UIStat
     protected open val layout: Int = 0
 
     abstract fun onBind()
-    open fun onRefresh(){}
+
+    open fun onRefresh() {}
```

Comment:
I think onRefresh() should be in interface

Reply:
why? what would be benefits?

Reply:
you have all that kind of functions inside the interface, because interface is a helper where you have functions